modules/services/picom.nix

```diff
@@ -41,6 +41,7 @@ let
       dnd           = { shadow = ${toJSON (!cfg.noDNDShadow)}; };
       popup_menu    = { opacity = ${cfg.menuOpacity}; };
       dropdown_menu = { opacity = ${cfg.menuOpacity}; };
+      ${cfg.extraWintypes}
```

Comment:
Probably better would be to deprecate these `menuOpacity`, `noDNDShadow`, `noDockShadow` settings and build `wintypes` from the list of attrsets?

Reply:
Oh yeah that's true, I think that's a good idea, I'll fix that soon

Reply:
Could this be done in a new PR so we can merge the current fixes?